src/components/sections/BannerText/banner-text.scss

```
@@ -1,44 +1,100 @@
 @import "src/styles/scaffold";
 
-[data-store-banner] {
-  padding-top: var(--fs-spacing-6);
-  padding-bottom: var(--fs-spacing-6);
-  text-align: center;
-  background-color: var(--fs-color-main-4);
+[data-fs-banner-text] {
+  // --------------------------------------------------------
+  // Design Tokens for Banner Text
+  // --------------------------------------------------------
+
+  --fs-banner-text-padding-mobile              : var(--fs-spacing-6) 0;
+  --fs-banner-text-padding-desktop             : var(--fs-spacing-9) 0;
+
+  --fs-banner-text-bkg-color                   : var(--fs-color-main-4);
+  --fs-banner-text-text-color                  : var(--fs-color-text-inverse);
+
+  --fs-banner-text-link-width                  : 11.25rem;
+  --fs-banner-text-link-padding                : var(--fs-spacing-2) var(--fs-spacing-4);
+  --fs-banner-text-link-margin-mobile          : var(--fs-spacing-5) 0 0 0;
+  --fs-banner-text-link-margin-desktop         : var(--fs-spacing-7) 0 0 0;
```

Shouldn't it be only `margin-top`?
(You should also change it on lines 53 and 57)
```suggestion
  --fs-banner-text-link-margin-top-mobile          : var(--fs-spacing-5);
  --fs-banner-text-link-margin-top-desktop         : var(--fs-spacing-7);
```